tron/api/resource.py

```diff
@@ -133,8 +155,8 @@ def getChild(self, action_name, _):
             action_run = self.job_run.action_runs[action_name]
             return ActionRunResource(action_run, self.job_run)
 
-        msg = "Cannot find action %s for %s"
-        return resource.NoResource(msg % (action_name, self.job_run))
+        msg = "Cannot find action '%s' for '%s'"
```

Comment:
Yuck. Just fstring this thing and merge with 159

Reply:
Oops, should've caught this.